frontend/src/component/project/Project/ExperimentalProjectFeatures/ExperimentalProjectTable/ExperimentalProjectTable.tsx

```diff
@@ -181,6 +184,9 @@ export const ExperimentalProjectFeatureToggles = ({
                         onChange={row?.getToggleSelectedHandler()}
                     />
                 ),
+                meta: {
+                    width: '1%',
```

Comment:
I'm not sure what this 1% is

Reply:
When we are using `display:table` instead of `flex`, it is "use as little space as possible"